content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,206 @@ aliases:
+
+| Metric Name       | Tag Values             | Aggregations                            |
+| ----------------- | ---------------------- | --------------------------------------- |
+| `request.Latency` | `host:A`, `endpoint:X` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:A`, `endpoint:Y` | `count`, `sum`, `min`, `max`, and `avg` |
+
+[1]: /developers/metrics/types/?tab=histogram
+{{% /tab %}}
+{{% tab "Distribution" %}}
+
+**A DISTRIBUTION metric generates five custom metrics** for each unique combination of metric name, host, and tag values to support the server-side aggregations  `count`, `sum`, `min`, `max`, and `avg`. [Learn more about DISTRIBUTION metric type][1].
+
+Suppose you have a distribution metric measuring `request.Latency` in which the following tag value combinations are present in your data as shown below:
+
+{{< img src="account_management/billing/custom_metrics/request_latency.png" alt="Request latency" responsive="true" style="width:80%;">}}
+
+The number of unique combinations of metric name, host, and tag values submitted for a `GAUGE` metric with this tagging scheme is 4:
+
+| Metric Name       | Tag Values                           |
+| ----------------- | ------------------------------------ |
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` |
+
+Given `request.Latency` is submitted as a `DISTRIBUTION`, Datadog [stores 5 custom metrics][1] for each of the original 4 unique tag value combinations to account for server side-aggregations: `count`, `sum`, `min`, `max`, and `avg`. This results in a total of 20 custom metrics for this `DISTRIBUTION` metric:
+
+| Metric Name       | Tag Values                           | Aggregations                            |
+| ----------------- | ------------------------------------ | --------------------------------------- |
+| `request.Latency` | `host:A`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:X`, `status:400` | `count`, `sum`, `min`, `max`, and `avg` |
+| `request.Latency` | `host:B`, `endpoint:Y`, `status:200` | `count`, `sum`, `min`, `max`, and `avg` |
+
+##### Distribution Metrics with Additional Percentile Aggregations
+
+[Adding additional percentile aggregations][2]: `p50`, `p75`, `p90`, `p95`, `p99`) to your DISTRIBUTION metric increases the number of distinct custom metrics by raising the ammount of aggregations. The final number of metric is:
+
```

Comment:
Add the following: 

Suppose the distribution metric, `request.Latency`, submitted the following values: 
** ADD EXAMPLE 6 TABLE HERE** 

For other metric types, like gauges and counts, we can combine/aggregate the most granular data to obtain globally accurate results. Percentiles can’t be recombined; therefore, they’re precomputed and stored individually.

You CAN’T calculate the globally accurate median across all hosts by combining/aggregating the median value on host:A with the median value on host:B:
(NOT CORRECT) median (the median on host:A, the median on host:B) 
	= median (100ms, 200ms) 
	= 150ms.

The global median request latency (175 ms) in your entire environment needs to be calculated off all data values:
(CORRECT)  median(100ms, 150ms, 200ms, 10s) =175 ms

To provide you with globally accurate percentiles, **Datadog stores 5 custom metrics per every potentially queryable tag value combination in your data.** 

Suppose you've added percentile aggregations for request.Latency for the set of tags {`endpoint`, `status`} with the same tag dependency seen earlier:

Datadog stores 5 custom metrics for each potentially queryable tag value combination listed below:
Potentially Queryable Tag Value Combinations:

- request.Latency{endpoint:X,status:200}
- request.Latency{endpoint:X,status:400}
- request.Latency{endpoint:Y,status:200} 
- request.Latency{endpoint:X}
- request.Latency{endpoint:Y} 
- request.Latency{status:200}
- request.Latency{status:400}
- request.Latency{*}

Note: We only count custom metrics  for queryable tag value combinations that actually appear in your data.  Since `request.Latency{endpoint:Y, status:400}` was never submitted in your data, this combination won’t be queryable and won’t count towards your custom metric count.